docs/content/developers/konnector/controllers/cluster/apiservicebinding.md

```diff
@@ -20,12 +20,21 @@ flowchart TD
     set_apiservicebinding_condition_connected_to_true(["Set condition 'Connected' to true"])
     set_apiservicebinding_condition_connected_to_false(["Set condition 'Connected' to false"])
     set_apiservicebinding_condition_schemainsync_to_true(["Set condition 'SchemaInSync' to true"])
-    set_apiservicebinding_condition_schemainsync_to_false(["Set condition 'SchemaInSync' to false"])
 
     get_apiserviceexport(["Get APIServiceExport"])
-    convert_apiserviceexport_to_crd(["Convert APIServiceExport to CRD"])
     is_apiserviceexport_present{"export<br>exists?"}
-    is_apiserviceexport_valid{"export<br>valid?"}
+
+    get_bound_schemas(["Get all referenced
```

Comment:
Double check please where does boundschema is created now. On request path or on request implementation. And source now has to play where it comes from

Reply:
This includes only changes from `servicebinding` and `serviceexport` reconcilers.
We can also describe, if it is consumer or provider side